__tests__/api.test.js

```diff
@@ -42,7 +42,73 @@ describe("GET /api/topics", () => {
 }),
 
 describe('GET api/articles', () => {
+  describe('successful connection tests', () => {
+    test('200: articles page returns an array containing objects of all articles', () => {
+      return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+          expect(Array.isArray(body.articles)).toBe(true)
+        })
+    })
+    test('200: all articles should have their given properties as listed below', () => {
+      return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({
+          body
+        }) => {
+          expect(body.articles.length).toBeGreaterThan(0)
```

Comment:
If you want to, you could assert a specific number in the length here as we know how many items are in the test database. Up to you though really.